pgjdbc/src/test/java/org/postgresql/test/jdbc3/StringTypeParameterTest.java

```diff
@@ -17,15 +22,11 @@
 import java.sql.Types;
 import java.util.Properties;
 
-public class StringTypeParameterTest extends TestCase {
+public class StringTypeParameterTest {
 
   private Connection _conn;
 
-  public StringTypeParameterTest(String name) {
-    super(name);
-  }
-
-  protected boolean setUp(String stringType) throws Exception {
+  private boolean prepare(String stringType) throws Exception {
```

Comment:
This might be a case for `@Parameterized`. On the other hand, I'm fine to keep the test as is.